src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery balls = new Lottery();
+        for (int i = 0; i < 3; i++) {
+            System.out.println(balls.getRandomBall().toString());
```

Comment:
toString method is automatically called by System.out.println on passed objects, no need to explicitly add it
```suggestion
            System.out.println(lottery.getRandomBall());
```